kajas/src/app/forgot-password/forgot-password.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+    <h1>Forgot Password</h1>
+    <div class="logo">
+      <img src="../../assets/kajas_black.png" alt="Kajas Logo">
+    </div>
+    <h3>"No worries, we got you covered!"</h3>
+    <form [formGroup]="forgotPasswordForm" (ngSubmit)="onSubmit()">
+        <div class="form-group-wrapper">
+            <div class="form-group">
+                <input type="email" id="user_email" formControlName="email" 
+                    placeholder="Email Address">
+                    @if(forgotPasswordForm.get('email').touched && 
```

Comment:
### For template control flow, observe proper indentation
- Try to find a [solution](https://github.com/angular/angular/issues/52720) online or uninstall prettier and format your codes manually.
https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g270ee9174f4_0_15